### PR TITLE
LPD-40181 Add tests

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-page-management** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by ealonso** [comment]
@sandrodw3 @georgel-pop-lr What happen with this one?

**Comment by sandrodw3** [comment]
> @sandrodw3 @georgel-pop-lr What happen with this one?

@georgel-pop-lr needs to test it to see if I missed some case

**Comment by georgel-pop-lr** [comment]
> > @sandrodw3 @georgel-pop-lr What happen with this one?
> 
> @georgel-pop-lr needs to test it to see if I missed some case

I will do it now

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/isMovementValid.ts [hunk 1]
```diff
@@ -154,3 +161,55 @@ export function isMovementValid({
...
+		return target.children[0];
+	}
+
+	// Return parent id if the item is not a valid paste target
+
+	if (!VALID_PASTE_TYPES.includes(target.type)) {
+		return target.parentId;
+	}
+
+	// Otherwise return the if of the item itself
```
Comment (georgel-pop-lr) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/isMovementValid.ts:212]: Do we miss some word on this comment?

Comment (georgel-pop-lr) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/isMovementValid.ts:212]: Okay, do that change, and I think this is good to be merged as is covering all the test cases.

**Comment by sandrodw3** [comment]
ci:test:page-management

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**